templates/projects/empty-server/files/server/server.js

```diff
@@ -22,8 +21,8 @@ app.start = function() {
 // Sub-apps like REST API are mounted via boot scripts.
 boot(app, __dirname, function(err) {
   if (err) throw err;
-
   // start the server if `$ node server.js`
-  if (require.main === module)
-    app.start();
+  if (require.main === module) {
+    var server = app.start();
```

Comment:
@raymondfeng appmetrics needs its reference, but I think we can handle it via `loopback-bluemix`.